analyzers/tests/SonarAnalyzer.UnitTest/TestCases/InitializeStaticFieldsInline.cs

```diff
@@ -160,4 +161,15 @@ public class ReproSwitchMulti
             }
         }
     }
+
+    public static class TestUtil
+    {
+        static TestUtil() // Noncompliant - FP, there are no static fields, see: https://github.com/SonarSource/sonar-dotnet/issues/6343
```

Comment:
To give the issue link higher change for surviving.
```suggestion
        // https://github.com/SonarSource/sonar-dotnet/issues/6343
        static TestUtil() // Noncompliant - FP, there are no static fields
```